Userland/Libraries/LibGfx/JPEGLoader.cpp

```diff
@@ -313,7 +313,13 @@ static inline i32* get_component(Macroblock& block, unsigned component)
 
 static ErrorOr<void> add_dc(JPEGLoadingContext& context, Macroblock& macroblock, ScanComponent const& scan_component)
 {
-    auto& dc_table = context.dc_tables.find(scan_component.dc_destination_id)->value;
```

Comment:
As you're likely aware, you could also do:

```
    auto dc_table_iterator = context.dc_tables.find(scan_component.dc_destination_id);
    if (dc_table_iterator == context.dc_tables.end()) {
        // error
    }
    auto& dc_table = dc_table_iterator->value;
```

Then it's more like the lhs. But as-is is fine too.

Reply:
As `dc_table_iterator == context.dc_tables.end()` is what `get` do. I think that it's what we should call